packages/perseus-utils/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@khanacademy/perseus-utils",
+    "description": "Miscellaneous utilities used by the Khan Academy Perseus ecosystem",
+    "author": "Khan Academy",
+    "license": "MIT",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Khan/perseus.git",
+        "directory": "packages/perseus-utils"
+    },
+    "bugs": {
+        "url": "https://github.com/Khan/perseus/issues"
```

Comment:
Random thought: do we even have the issues tab enabled?

Reply:
We do! Not sure that's good or bad. I'll leave this here for now. :)